goodtablesio/tests/plugins/github/test_tasks.py

```diff
@@ -1,17 +1,24 @@
 from unittest.mock import patch
 from goodtablesio.plugins.github import tasks
 
+# Constants
+
+OWNER = 'frictionlessdata'
+REPO = 'goodtables.io-example'
+SHA = 'd5be243487d9882d7f762e7fa04b36b900164a59'
+
 
 # Tests
 
 def test_get_job_base():
-    actual = tasks._get_job_base('frictionlessdata', 'goodtables.io-example')
-    expect = 'https://raw.githubusercontent.com/frictionlessdata/goodtables.io-example/master'  # noqa
+    actual = tasks._get_job_base(OWNER, REPO, SHA)
+    expect = 'https://raw.githubusercontent.com/frictionlessdata/'
+    expect += 'goodtables.io-example/d5be243487d9882d7f762e7fa04b36b900164a59'
     assert actual == expect
 
 
 def test_get_job_files():
-    actual = tasks._get_job_files('frictionlessdata', 'goodtables.io-example', 'd5be243487d9882d7f762e7fa04b36b900164a59')  # noqa
+    actual = tasks._get_job_files(OWNER, REPO, SHA)
```

Comment:
Yes. But for now just don't see a reason for it - tests are fast, limits are ok. Any mocking layer reduces confidence that it really works. I think we should rebase on httpretty when it will start to be slow.